docs/tools-and-applications/mlflow.md

```diff
@@ -0,0 +1,55 @@
+# MLFlow
+MLflow is an open source platform to manage the ML lifecycle, including experimentation, reproducibility, deployment, and a central model registry. MLflow currently offers four components:
+![The 4 capabilities of MLFlow. Source: https://mlflow.org/](img/MLFlow_capabilities.PNG)
+Read more here: [https://mlflow.org/](https://mlflow.org/)
+
+## Helm installation into OpenShift namespace
+**Pre-requisites**  
+
```

Comment:
Briefly explain why: ODF to provide S3 storage, and Postgre to store MLFlow config. That will people to understand and eventually change them.

Reply:
Added some brief texts